benchexec/intel_cpu_energy.py

```diff
@@ -0,0 +1,86 @@
+import warnings
```

Comment:
Here our typical header with license and `from future import` is missing. There should also be a comment like in runexecutor that this module needs to support Python 2.